macaroons/service.go

```diff
@@ -16,7 +22,7 @@ var (
 
 // NewService returns a service backed by the macaroon Bolt DB stored in the
 // passed directory.
-func NewService(dir string) (*bakery.Service, error) {
+func NewService(dir string, checks ...Checker) (*bakery.Bakery, error) {
```

Comment:
`godoc` comment could use a section explaining any expectations that the `checks` arg should have before passing it into the constructor.

Reply:
Updated the comment to clarify the `checks` argument.